python/ray/autoscaler/command_runner.py

```diff
@@ -143,7 +145,10 @@ def run(
         """
         raise NotImplementedError
 
-    def run_rsync_up(self, source: str, target: str) -> None:
+    def run_rsync_up(self,
+                     source: str,
+                     target: str,
+                     options: Dict[str, Any] = {}) -> None:
```

Comment:
```suggestion
                     options: Dict[str, Any] = None) -> None:
```
Be very careful - that default '{}' can be mutated at runtime.

Reply:
Oh YIKES! I totally forgot about this "feature" of python.